src/lib/snark_worker/functor.ml

```diff
@@ -344,7 +350,7 @@ module Make (Inputs : Intf.Inputs_intf) :
         flag "--daemon-address" ~aliases:[ "daemon-address" ]
           (required (Arg_type.create Host_and_port.of_string))
           ~doc:"HOST-AND-PORT address daemon is listening on"
-      and cli_proof_level =
+      and proof_level =
```

Comment:
proof level can be set from the command line and ovveride the compile time config entry.